vnext/Desktop.ABITests/ReactNativeHostTests.cpp

```diff
@@ -0,0 +1,26 @@
+#include "pch.h"
```

Comment:
>#i [](start = 0, length = 2)

nit: please add the copyright header.

Reply:
Fixed.

---
In reply to: [423291311](https://github.com/microsoft/react-native-windows/pull/4848#discussion_r423291311) [](ancestors = 423291311)